crates/re_viewer/src/misc/caches/tensor_image_cache.rs

```diff
@@ -486,6 +451,18 @@ impl AsDynamicImage for ClassicTensor {
 
                 Ok(DynamicImage::ImageRgb8(img))
             }
+
+            (_depth, dtype, meaning @ TensorDataMeaning::ClassId) => {
+                anyhow::bail!(
+                    "Shape={shape:?} and dtype={dtype:?} is incompatible with meaning={meaning:?}"
+                )
+            }
+
+            (_depth, dtype, _) => {
+                anyhow::bail!(
+                                "Don't know how to turn a tensor of shape={shape:?} and dtype={dtype:?} into an image"
+                            )
```

Comment:
weird formatting

Reply:
Weird -- surprised cargo fmt did that and then preserved it.